packages/dapper/src/cyberstormSchemas/achievement.ts

```diff
@@ -0,0 +1,16 @@
+import { z } from "zod";
+
+export const achievementSchema = z.object({
+  name: z.string().min(1),
+  namespace: z.string().min(1),
+  imageSource: z.string(),
+  description: z.string(),
+});
+
+export const achievementSettingSchema = z.object({
```

Comment:
https://zod.dev/?id=extend

Reply:
You know, when I went to bed, it occured to me, that there might've been an extend function 🤠